### PR TITLE
Transfer Files - Add report usage SH size and service id

### PR DESCRIPTION
**Pull request by omerzi**
- [x] All [tests](https://github.com/jfrog/jfrog-cli-core#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [x] All [static analysis checks](https://github.com/jfrog/jfrog-cli-core/actions/workflows/analysis.yml) passed.
- [x] This pull request is on the dev branch.
- [x] I used gofmt for formatting the code before submitting the pull request.
-----
 Based on: https://github.com/jfrog/jfrog-client-go/pull/649

### CODE REVIEW

File: artifactory/commands/transferfiles/transfer.go [hunk 1]
```diff
@@ -130,6 +133,37 @@ func (tdc *TransferFilesCommand) Run() (err error) {
...
+		},
+		{
+			AttributeName:  "sourceStorageSize",
+			AttributeValue: sourceStorageInfo.BinariesSize,
+		},
+	}
+	err = usage.SendReportUsage(coreutils.GetCliUserAgent(), tdc.CommandName(), tdc.targetStorageInfoManager.GetServiceManager(), reportUsageAttributes...)
+	if err != nil {
+		log.Debug(err.Error())
+		return
```
Comment (yahavi) [inline artifactory/commands/transferfiles/transfer.go:163]: Redundant return
```suggestion
```

File: artifactory/commands/transferfiles/transfer.go [hunk 2]
```diff
@@ -130,6 +133,37 @@ func (tdc *TransferFilesCommand) Run() (err error) {
 	return tdc.cleanup(err, allSourceLocalRepos)
 }
 
+func (tdc *TransferFilesCommand) ReportTransferFilesUsage() {
```
Comment (yahavi) [inline artifactory/commands/transferfiles/transfer.go:136]: Can be private:
```suggestion
func (tdc *TransferFilesCommand) reportTransferFilesUsage() {
```

**Comment by yahavi** [comment]
Please also label this PR

File: artifactory/commands/transferfiles/transfer.go (hunk 1 above)
Comment (omerzi) [reply artifactory/commands/transferfiles/transfer.go:163]: done

File: artifactory/commands/transferfiles/transfer.go [hunk 3]
```diff
@@ -130,6 +133,37 @@ func (tdc *TransferFilesCommand) Run() (err error) {
...
+	var err error
+	log.Debug(usage.ReportUsagePrefix + "Sending Transfer File info...")
```
[the remark this answers is not in the document]
Comment (omerzi) [reply artifactory/commands/transferfiles/transfer.go:138]: done

File: artifactory/commands/transferfiles/transfer.go [hunk 4]
```diff
@@ -130,6 +133,37 @@ func (tdc *TransferFilesCommand) Run() (err error) {
 	return tdc.cleanup(err, allSourceLocalRepos)
 }
 
+func (tdc *TransferFilesCommand) ReportTransferFilesUsage() {
+	var err error
```
[the remark this answers is not in the document]
Comment (omerzi) [reply artifactory/commands/transferfiles/transfer.go:137]: done

File: artifactory/commands/transferfiles/transfer.go (hunk 2 above)
Comment (omerzi) [reply artifactory/commands/transferfiles/transfer.go:136]: done

**Review by eyalbe4 — APPROVED** [review]

**Pull request merged**